project.clj

```diff
@@ -1,3 +1,58 @@
+(require 'clojure.java.io)
```

Comment:
I'd add some section comments around this, explaining the need for it, so casual readers of the project's code are not surprised.